src/ethereum/test-driver.ts

```diff
@@ -0,0 +1,125 @@
+import { Driver } from '@orbs-network/orbs-ethereum-contracts-v2';
+import {
+    evmIncreaseTime,
+    evmMine,
+    getTopBlockTimestamp,
+} from '@orbs-network/orbs-ethereum-contracts-v2/release/test/helpers';
+import { MonthlySubscriptionPlanContract } from '@orbs-network/orbs-ethereum-contracts-v2/release/typings/monthly-subscription-plan-contract';
+
+const SCENARIO_MAX_STANDBYS = 1;
+const SCENARIO_MAX_COMMITTEE_SIZE = 2;
+const SUBSCRIPTION_MONTHLY_RATE = 1000;
+const SECONDS_IN_MONTH = 30 * 24 * 60 * 60;
+
+export class EthereumTestDriver {
+    private orbsPosV2Driver?: Driver;
+    private subscriber?: MonthlySubscriptionPlanContract;
+
+    constructor(public verbose = false) {}
+
+    async deployContracts() {
+        if (this.verbose) console.log(`[posv2] about to deploy contracts`);
+        this.orbsPosV2Driver = await Driver.new({
+            maxStandbys: SCENARIO_MAX_STANDBYS,
+            maxCommitteeSize: SCENARIO_MAX_COMMITTEE_SIZE,
+        });
+        if (this.verbose) console.log(`[posv2] about to deploy subscriber`);
+        this.subscriber = await this.orbsPosV2Driver.newSubscriber('defaultTier', SUBSCRIPTION_MONTHLY_RATE);
+    }
+
+    getContractRegistryAddress(): string {
+        if (!this.orbsPosV2Driver) throw new Error(`Driver contracts not deployed`);
+        const d = this.orbsPosV2Driver;
+
+        const res = d.contractRegistry.address;
+        if (this.verbose) console.log(`[posv2] contract registry address is ${res}`);
+        return res;
+    }
+
+    async setupInitialCommittee() {
+        if (!this.orbsPosV2Driver) throw new Error(`Driver contracts not deployed`);
```

Comment:
lol, it fixes lint errors :)
Annoying Typescript shit.
The type of `this.orbsPosV2Driver` is nullable because if somebody didn't run deploy it may be null and the typescript linter doesn't like when we access nullable types without checking first. I figured it's nicer to add an explicit test than shut the linter up via config.